### ISSUE TITLE
Node equivalent?

### ISSUE THREAD

**Issue by natew** [GoogleChromeLabs/clooney#125]
This may be a strange place to ask but some searching hasn't been fruitful yet, but this exact interface for Node would be quite helpful to me. Anyone know of a similar library?

**Comment by surma** [comment]
I am not aware of any projects for Node that do something like Clooney.

I have a TODO to make Comlink work with Node. Once that happens, Clooney should “just work” with Node.

**Comment by yoDon** [comment]
Any chance you've had a moment or two to think about a Node release? Wishing I could use this on the server.

**Comment by trusktr** [comment]
Looks like there's workers in Node now: https://nodejs.org/api/worker_threads.html

Looks like Comlink just needs to look for `on`/`off` instead of `addEventListener`/`removeEventListener`?

**Comment by surma** [comment]
I was waiting for [this PR](https://github.com/nodejs/node/pull/26082) to land. It has landed and got merged into Node 11.11 it seems, but neither Node 11.11 nor Node 11.12 exhibit the desired behavior. 

Once that lands, I’ll add support for Node in Comlink v4, which should just be an additional endpoint wrapper.

**Comment by yoDon** [comment]
It looks like you have Comlink working in node - anything we need to do to enable Clooney support in node?
